modules/ethereum/src/mock.rs

```diff
@@ -182,3 +183,12 @@ pub fn insert_header<S: Storage>(storage: &mut S, header: Header) {
 		finality_votes: FinalityVotes::default(),
 	});
 }
+
+/// Pruning strategy that keeps 10 headers behind best block.
+pub struct Keep10HeadersBehindBest;
```

Comment:
By making `PruningStrategy` work with `&mut self` we could make parametrisation of this type easier (i.e. just pass `10` as parameter). `Kovan` one would be just a wrapper in such case.

Reply:
Kovan keeps some blocks behind best finalized && this keeps some blocks behind best known block.